components/utils/cachedrequestloader/src/main/java/org/mozilla/cachedrequestloader/BackgroundCachedRequestLoader.java

```diff
@@ -29,6 +30,9 @@
     private int socketTag;
     private ResponseData stringLiveData;
     private static final String TAG = "CachedRequestLoader";
+    // Mainly for testing purposes.
+    private boolean delayCacheLoad = false;
```

Comment:
can we add @VisibleForTesting for this?

Reply:
They are already private. I'm not sure if that makes sense.

Reply:
Fixed.